### PR TITLE
Add --ca-cert arg to encrypt-ami

### PR DESCRIPTION
**Pull request by bonzotronix**
* Adds --ca-cert arg to 'encrypt-ami' command
  (TODO: add it to update-ami, encrypt-gce-image and update-gce-image)

[rebased and fixed version of #322, which superseded #308]

### CODE REVIEW

File: brkt_cli/aws/__init__.py [hunk 1]
```diff
@@ -358,6 +361,25 @@ def command_encrypt_ami(values, log):
...
+        try:
+            with open(values.ca_cert, 'r') as f:
+                ca_cert_data = f.read()
+        except IOError as e:
+            raise ValidationError(e)
+        try:
+            parsed_cert = x509.load_pem_x509_certificate(ca_cert_data,
+                                                         default_backend())
+        except Exception as e:
+            raise ValidationError('Error validating CA cert: %s', str(e))
```
Comment (bburtin) [inline brkt_cli/aws/__init__.py]: I think you want `%`, not `,`

Comment (bonzotronix) [inline brkt_cli/aws/__init__.py]: Right on both counts.  Will fix.

Comment (bonzotronix) [inline brkt_cli/aws/__init__.py]: Fixed in new commit.  PTAL.

**Comment by bburtin** [comment]
:shipit:

**Pull request merged**